README.md

```diff
@@ -0,0 +1,69 @@
+# OCP DNSNameResolver
+
+## Name
+
+OCP DNSNameResolver - updates the status of the [DNSNameResolver](https://github.com/openshift/api/blob/master/network/v1alpha1/types_dnsnameresolver.go)
+custom resources with IP addresses of matching resolved DNS names.
+
+## Description
+
+The spec of DNSNameResolver custom resource takes as input a DNS name. The DNS name can be either a regular or a wildcard DNS name. The plugin intercepts
+the DNS lookups of the matching DNS names and updates the status of these CRs with the IP addresses of the matching resolved names.
+
+The prerequisite for enabling this plugin are:
```

Comment:
The prerequisites may deserve a dedicated chapter.

Reply:
Normally the sections in the Readme of CoreDNS plugins are Name, Description, Syntax, Examples, etc. The Description section contains all the details about the plugin and how to deploy. That's why I added the prerequisites here. https://coredns.io/explugins/